kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -68,10 +68,14 @@ def __init__(self, name, init_config, instances):
             self.instance.get('monitor_all_broker_highwatermarks', False)
         )
         self._consumer_groups = self.instance.get('consumer_groups', {})
-        self._kafka_client = self._create_kafka_admin_client()
```

Comment:
We intentionally keep an open connection for performance reasons. Let's keep it here